packages/dds/sequence/api-report/sequence.legacy.alpha.api.md

```diff
@@ -297,177 +297,56 @@ export { reservedMarkerIdKey }
     readonly isLocal: boolean;
     // (undocumented)
     readonly opArgs: IMergeTreeDeltaOpArgs;
 }
 
 // @alpha
-export abstract class SequenceEvent<TOperation extends MergeTreeDeltaOperationTypes = MergeTreeDeltaOperationTypes> {
-    // @deprecated
-    constructor(
-    deltaArgs: IMergeTreeDeltaCallbackArgs<TOperation>, mergeTreeClient: Client);
-    get clientId(): string | undefined;
+export interface SequenceEvent<TOperation extends MergeTreeDeltaOperationTypes = MergeTreeDeltaOperationTypes> {
+    readonly clientId: string | undefined;
+    // (undocumented)
     readonly deltaArgs: IMergeTreeDeltaCallbackArgs<TOperation>;
     // (undocumented)
     readonly deltaOperation: TOperation;
-    get first(): Readonly<ISequenceDeltaRange<TOperation>>;
-    get last(): Readonly<ISequenceDeltaRange<TOperation>>;
-    get ranges(): readonly Readonly<ISequenceDeltaRange<TOperation>>[];
+    readonly first: Readonly<ISequenceDeltaRange<TOperation>>;
+    readonly last: Readonly<ISequenceDeltaRange<TOperation>>;
+    readonly ranges: readonly Readonly<ISequenceDeltaRange<TOperation>>[];
 }
 
 // @alpha
-export class SequenceInterval implements ISerializableInterval {
-    // @deprecated
-    constructor(client: Client,
-    start: LocalReferencePosition,
-    end: LocalReferencePosition, intervalType: IntervalType, props?: PropertySet, startSide?: Side, endSide?: Side);
-    addPositionChangeListeners(beforePositionChange: () => void, afterPositionChange: () => void): void;
-    // (undocumented)
-    addProperties(newProps: PropertySet, collab?: boolean, seq?: number): PropertySet | undefined;
-    // (undocumented)
-    clone(): SequenceInterval;
-    compare(b: SequenceInterval): number;
-    compareEnd(b: SequenceInterval): number;
-    compareStart(b: SequenceInterval): number;
-    end: LocalReferencePosition;
+export interface SequenceInterval extends ISerializableInterval {
+    readonly end: LocalReferencePosition;
     // (undocumented)
     readonly endSide: Side;
-    getIntervalId(): string;
     // (undocumented)
-    intervalType: IntervalType;
-    modify(label: string, start: SequencePlace | undefined, end: SequencePlace | undefined, op?: ISequencedDocumentMessage, localSeq?: number, useNewSlidingBehavior?: boolean): SequenceInterval;
-    // (undocumented)
-    overlaps(b: SequenceInterval): boolean;
-    // (undocumented)
-    overlapsPos(bstart: number, bend: number): boolean;
-    properties: PropertySet;
-    // (undocumented)
-    propertyManager: PropertiesManager;
-    removePositionChangeListeners(): void;
+    readonly intervalType: IntervalType;
     // (undocumented)
-    serialize(): ISerializedInterval;
-    start: LocalReferencePosition;
+    readonly start: LocalReferencePosition;
     // (undocumented)
     readonly startSide: Side;
     // (undocumented)
-    get stickiness(): IntervalStickiness;
-    union(b: SequenceInterval): SequenceInterval;
+    readonly stickiness: IntervalStickiness;
```

Comment:
There are a bunch of changes here that are not to deprecated methods/types.
This includes a very serious type relaxing that means implementation will fail to work properly unless the input matches the more strict expectation.
(The SequenceInterval class overrides IInterval to say that all intervals it works with are SequenceIntervals, which typescript unfortunately allows. This restriction should really be preserved.)